tests/conftest.py

```diff
@@ -123,7 +116,7 @@ def pytest_addoption(parser):
 @pytest.fixture
 def should_generate_snapshots(pytestconfig):
     """Indicates whether snapshots have been instructed to be generated."""
-    return bool(pytestconfig.getoption("generate_snapshots"))
+    return True
```

Comment:
Why do we generate snapshots unconditionally now?

Reply:
Leftover from some tests I was running. Fixing it